src/main/java/httpserver/HTTPResponse.java

```diff
@@ -0,0 +1,22 @@
+package httpserver;
+
+public class HTTPResponse {
+
+    private String statusCode;
+
+    public HTTPResponse(int statusCode) {
+        this.statusCode = String.valueOf(statusCode);
+    }
+
+    public String HTTP() {
+        return "HTTP/1.1";
+    }
+
+    public String getStatusCode() {
+        return statusCode;
+    }
+
+    public String getReasonPhrase() {
+        return "OK";
```

Comment:
I assume it's temporary alway the same answer (?) - may be use a enum for the different type of answer